services/setup/lib/paths-list/index.js

```diff
@@ -1,9 +1,21 @@
+const fs = require('fs');
 const { readDir } = require('../fs');
+const pathLib = require('path');
+
+const isDirectory = (rootPath) => (path) => (
+  fs.statSync(pathLib.join(rootPath, path)).isDirectory()
+);
+
+const isNotHidden = (path) => path[0] !== ".";
 
 const pathsList = async ({ rootPath, ignore = [] }) => {
   const paths = await readDir(rootPath);
+  console.log(rootPath, paths);
```

Comment:
I like to leave `console.log`s in as I usually only scroll through the output when I'm trying to figure out why it's not working. 

But, I know that you'd prefer to remove them so I thought I'd point this one out :-D